spec/controllers/users/confirmations_controller_spec.rb

```diff
@@ -9,7 +9,7 @@
     it 'tracks nil email confirmation token' do
       analytics_hash = {
         success: false,
-        error: "Confirmation token can't be blank",
+        error: 'Confirmation token Please fill in this field.',
```

Comment:
ah, yeah, this is not user facing, so a little wording awkwardness here is fine i think